lib/index.js

```diff
@@ -16,30 +16,45 @@ var Yandex = module.exports = integration('Yandex Metrica')
   .assumesPageview()
   .global('yandex_metrika_callbacks')
   .global('Ya')
+  .option('type', 0)
   .option('counterId', null)
   .option('clickmap', false)
   .option('webvisor', false)
+  .option('trackHash', false)
+  .option('trackLinks', false)
+  .option('accurateTrackBounce', false)
+  .option('params', null)
```

Comment:
@francisco Sure. Is it possible to do something like passing data from `analytics.track()` or `analytics.identify()` Yandex.Metrika afterwards via `params`? But I think that would just duplicate the visit amount :/